src/convertToObject.js

```diff
@@ -3,16 +3,21 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';')
+    .filter((item) => item.trim() !== '')
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable.
![image](https://github.com/mate-academy/js_style-to-object/assets/63243136/f2976863-9efc-4e1f-8f48-737b03fcc5f8)
Also, in the filter you don't need to compare with an empty string you can just return `item.trim()` and all falsy values will be excluded. `.filter((item) => item.trim())`